<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1018</width>
    <height>727</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="m_toolBar" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>80</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">#m_toolBar{background-color: rgb(142, 6, 38);}

#labelPicture
{background-color: rgba(255, 255, 255, 0);
border-image: url(:/new/prefix1/m_icon/user.png);}
#labelPicture:hover{background-color: rgba(255, 255, 255, 0);
border-image: url(:/new/prefix1/m_icon/user-fill.png);}
#labelPicture:pressed{background-color: rgba(255, 255, 255, 0);
border-image: url(:/new/prefix1/m_icon/user-fill.png);}


#btnUser{background-color: rgba(255, 255, 255, 10);}



#btnMessage{background-color: rgba(255, 255, 255, 0);
border-image: url(:/new/prefix1/m_icon/bell.png);}
#btnMessage:hover{background-color: rgba(255, 255, 255, 0);
border-image: url(:/new/prefix1/m_icon/bell_fill.png);}
#btnMessage:pressed{background-color: rgba(255, 255, 255, 0);
border-image: url(:/new/prefix1/m_icon/bell_fill.png);}

#btnSet{background-color: rgba(255, 255, 255, 0);
border-image: url(:/new/prefix1/m_icon/setting.png);}
#btnSet:hover{background-color: rgba(255, 255, 255, 0);
border-image: url(:/new/prefix1/m_icon/setting-fill.png);}
#btnSet:pressed{background-color: rgba(255, 255, 255, 0);
border-image:  url(:/new/prefix1/m_icon/setting-fill.png);}



#btnMin{background-color: rgba(255, 255, 255, 0);
border-image: url(:/new/prefix1/m_icon/minus-circle.png);}
#btnMin:hover{background-color: rgba(255, 255, 255, 0);
border-image: url(:/new/prefix1/m_icon/minus-circle-fill.png);}
#btnMin:pressed{background-color: rgba(255, 255, 255, 0);
border-image: url(:/new/prefix1/m_icon/minus-circle-fill.png);}

#btnClose{background-color: rgba(255, 255, 255, 0);
border-image: url(:/new/prefix1/m_icon/close-circle.png);}
#btnClose:hover{background-color: rgba(255, 255, 255, 0);
border-image: url(:/new/prefix1//m_icon/close-circle-fill.png);}
#btnClose:pressed{background-color: rgba(255, 255, 255, 0);
border-image: url(:/new/prefix1/m_icon/close-circle-fill.png);}
</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="labelLogo">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image: url(:/new/prefix1/C:/Users/z/Desktop/icon_image/raven.png);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelPaltformName">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 9pt &quot;GothicE&quot;;</string>
          </property>
          <property name="text">
           <string>Z C_TRANSLATION</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background:transparent;
font: 25 7pt &quot;等线 Light&quot;;

border:1px groove white;
border-radius:20px;
padding:2px 4px;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>请输入搜索内容</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>118</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="labelPicture">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnUser">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnMessage">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSet">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="font">
           <font>
            <pointsize>6</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(183, 183, 183);</string>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnMin">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnClose">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="frame">
       <property name="styleSheet">
        <string notr="true">#frame{
	border-image: url(:/new/prefix1/m_icon/58a6bf16282d2.png);}

#btnAllAssign{
background-color: rgba(255, 255, 255, 0);}
#btnAllAssign:hover{
background-color: rgba(244, 244, 244, 80);}

#btnMyPub{
background-color: rgba(255, 255, 255, 0);}
#btnMyPub:hover{
background-color: rgba(244, 244, 244, 80);}

#btnMyManager{
background-color: rgba(255, 255, 255, 0);}
#btnMyManager:hover{
background-color: rgba(244, 244, 244, 80);}

#btnMyTran{
background-color: rgba(255, 255, 255, 0);}
#btnMyTran:hover{
background-color: rgba(244, 244, 244, 80);}

#btnMyReview{
background-color: rgba(255, 255, 255, 0);}
#btnMyReview:hover{
background-color: rgba(244, 244, 244, 80);}

#btnFindManager{
background-color: rgba(255, 255, 255, 0);}
#btnFindManager:hover{
background-color: rgba(244, 244, 244, 80);}

#btnFindTran{
background-color: rgba(255, 255, 255, 0);}
#btnFindTran:hover{
background-color: rgba(244, 244, 244, 80);}

#btnProgressing{
background-color: rgba(255, 255, 255, 0);}
#btnProgressing:hover{
background-color: rgba(244, 244, 244, 80);}

#btnPublish{
background-color: rgba(255, 255, 255, 0);}
#btnPublish:hover{
background-color: rgba(244, 244, 244, 80);}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QWidget" name="widget" native="true">
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout" stretch="10,10,8,8,8,1,10,1,10,1,10,1,10,10,0">
           <item>
            <widget class="QLabel" name="label">
             <property name="styleSheet">
              <string notr="true">font: 18pt &quot;宋体&quot;;</string>
             </property>
             <property name="text">
              <string>任务列表</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,10">
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="btnAllAssign">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>所有任务</string>
               </property>
               <property name="icon">
                <iconset resource="image.qrc">
                 <normaloff>:/new/prefix1/m_icon/assignment.png</normaloff>:/new/prefix1/m_icon/assignment.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,3">
             <item>
              <spacer name="horizontalSpacer_7">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="btnFindManager">
               <property name="text">
                <string>负责人招募</string>
               </property>
               <property name="icon">
                <iconset resource="image.qrc">
                 <normaloff>:/new/prefix1/m_icon/findMan.png</normaloff>:/new/prefix1/m_icon/findMan.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="1,3">
             <item>
              <spacer name="horizontalSpacer_8">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="btnFindTran">
               <property name="styleSheet">
                <string notr="true">font: 10pt &quot;宋体&quot;;</string>
               </property>
               <property name="text">
                <string>译者招募</string>
               </property>
               <property name="icon">
                <iconset resource="image.qrc">
                 <normaloff>:/new/prefix1/m_icon/findTran.png</normaloff>:/new/prefix1/m_icon/findTran.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="1,3">
             <item>
              <spacer name="horizontalSpacer_9">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="btnProgressing">
               <property name="text">
                <string>任务进行中</string>
               </property>
               <property name="icon">
                <iconset resource="image.qrc">
                 <normaloff>:/new/prefix1/m_icon/progressing.png</normaloff>:/new/prefix1/m_icon/progressing.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_6">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>17</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,10">
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="btnMyPub">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>我发布的任务</string>
               </property>
               <property name="icon">
                <iconset resource="image.qrc">
                 <normaloff>:/new/prefix1/m_icon/lancher.png</normaloff>:/new/prefix1/m_icon/lancher.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>17</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,10">
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="btnMyManager">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>我负责的任务</string>
               </property>
               <property name="icon">
                <iconset resource="image.qrc">
                 <normaloff>:/new/prefix1/m_icon/manager.png</normaloff>:/new/prefix1/m_icon/manager.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_5">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>17</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,10">
             <item>
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="btnMyTran">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>我翻译的任务</string>
               </property>
               <property name="icon">
                <iconset resource="image.qrc">
                 <normaloff>:/new/prefix1/m_icon/trans.png</normaloff>:/new/prefix1/m_icon/trans.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>18</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,10">
             <item>
              <spacer name="horizontalSpacer_6">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="btnMyReview">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>我审核的任务</string>
               </property>
               <property name="icon">
                <iconset resource="image.qrc">
                 <normaloff>:/new/prefix1/m_icon/review.png</normaloff>:/new/prefix1/m_icon/review.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,10">
             <item>
              <spacer name="horizontalSpacer_10">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="btnPublish">
               <property name="styleSheet">
                <string notr="true">font: 14pt &quot;宋体&quot;;</string>
               </property>
               <property name="text">
                <string>发布任务</string>
               </property>
               <property name="icon">
                <iconset resource="image.qrc">
                 <normaloff>:/new/prefix1/m_icon/launch.png</normaloff>:/new/prefix1/m_icon/launch.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidget">
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255, 255, 255, 0);
selection-background-color: rgba(255, 255, 255,255);
gridline-color: rgba(255, 255, 255, 0);</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1018</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="styleSheet">
    <string notr="true">QAciton#actionSearch{
border-image: url(:/new/prefix1/C:/Users/z/Desktop/icon_image/search.png);}
QAciton#actionSearch:hover{
border-image: url(:/new/prefix1/C:/Users/z/Desktop/icon_image/search.png);}</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);
border:0;</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionSearch">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/new/prefix1/m_icon/search_click.png</normaloff>:/new/prefix1/m_icon/search_click.png</iconset>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </action>
  <action name="actionBasicPreserved">
   <property name="icon">
    <iconset>
     <normaloff>m_icon/edit.png</normaloff>m_icon/edit.png</iconset>
   </property>
   <property name="text">
    <string>用户基本信息维护</string>
   </property>
  </action>
  <action name="actionRealNameAuth">
   <property name="icon">
    <iconset>
     <normaloff>m_icon/realName.png</normaloff>m_icon/realName.png</iconset>
   </property>
   <property name="text">
    <string>资质认证</string>
   </property>
  </action>
  <action name="actionLogOut">
   <property name="icon">
    <iconset>
     <normaloff>m_icon/log_out.png</normaloff>m_icon/log_out.png</iconset>
   </property>
   <property name="text">
    <string>注销</string>
   </property>
  </action>
  <action name="actionRecharge">
   <property name="icon">
    <iconset>
     <normaloff>m_icon/YUAN.png</normaloff>m_icon/YUAN.png</iconset>
   </property>
   <property name="text">
    <string>充值</string>
   </property>
  </action>
  <action name="actionResetCode">
   <property name="icon">
    <iconset>
     <normaloff>m_icon/lock.png</normaloff>m_icon/lock.png</iconset>
   </property>
   <property name="text">
    <string>更改密码</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>用户认证</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
